libs/zzz/formula-ui/src/formulaText.tsx

```
@@ -21,24 +26,29 @@ export function translate(
+  ): ReactNode[] {
     return ops.map((op) => {
-      const text = translate(op, cache)
-      if (text.name) return deps.add(text), text.name
+      const text = formulaText(op, cache)
+      if (text.name) {
+        deps.add(text)
+        return text.name
+      }
       text.deps.forEach((dep) => deps.add(dep))
-      return text.prec >= prec ? text.formula : `(${text.formula})`
+      return text.prec >= prec ? text.formula : <span>({text.formula})</span>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)